monolithic/monolithic-layered/src/test/java/com/epam/training/microservices/monolithic/service/doctor/RecipeRepositoryTest.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.training.microservices.monolithic.service.doctor;
+
+import com.epam.training.microservices.monolithic.model.recipie.Doctor;
+import com.epam.training.microservices.monolithic.model.recipie.Recipe;
+import com.epam.training.microservices.monolithic.model.recipie.Recipient;
+import com.epam.training.microservices.monolithic.service.DataJpaTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class RecipeRepositoryTest extends DataJpaTest {
+
+    @Autowired
+    private RecipeRepository repository;
+
+    @Autowired
+    private DoctorRepository doctorRepository;
+
+    @Autowired
+    private RecipientRepository recipientRepository;
+
+    @Test
+    public void shouldSaveAndRetrieveRecipe() {
+        // Arrange
+        Doctor doctor = new Doctor();
```

Comment:
Btw, if you don’t want to create complex entities, it is always possible to use `Doctor doctor = mock(Doctor.class)` or use a third-party library like EasyRandom. It is useful for scenarios when the data itself is not important and can be random.

Reply:
I keep this in mind. But in these tests and with these simple entities there's no big difference.

Reply:
Sure